### ISSUE TITLE
Add asynchronous database handling

### ISSUE THREAD

**Issue by kingkaos** [Bradyphrenia/biene#3]
Using asynchronous handling would allow app interaction while accessing database.

**Comment by Bradyphrenia** [comment]
Would be nice but there are not so much data to deal with.

My App does not compile well for any reason. So i have to use my little CLI :-)

**Comment by kingkaos** [comment]
You are right, but adding asynchronous handling would keep the application active while searching, writing or reading data. So ne freezing would of the app appears.

**Comment by kingkaos** [comment]
> My App does not compile well for any reason. So i have to use my little CLI :-) 

Could you please sent or post the error message so that I can review and fix the build? Just add a new issue 😊
